<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckPageItem</class>
 <widget class="QWidget" name="CheckPageItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>122</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-bottom: 1px solid #EBEDF0;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="ItemBg">
     <property name="styleSheet">
      <string notr="true">#ItemBg{

	border: 1px solid #EBEDF0;
}

*{
	font-family: '微软雅黑';
	font-size: 14px;
	color: #222222;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLineEdit" name="userId">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>150</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1234</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="nickNameLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>138</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>用户1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="videoTitleLabel">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>0</y>
        <width>265</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#videoTitleLabel{
	padding-left: 34px;
	padding-right: 34px;
	border: 1px solid #EBEDF0;


}</string>
      </property>
      <property name="text">
       <string>视频标题</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="videoBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>0</y>
        <width>200</width>
        <height>118</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#videoBtn{
	background-color: transparent;
	border: none;
	border-image: url(':/image/vedioFirstPage.png');
	border-bottom: 1px solid #EBEDF0;

} </string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="statusBtn">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>40</y>
        <width>72</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#statusBtn{
	background-color: #EBF3FF;
	border: none;
	border-radius: 10px;
	color: #3686FF;
	font-size: 12px;
	line-height: 16px;
}</string>
      </property>
      <property name="text">
       <string>已审核</string>
      </property>
     </widget>
     <widget class="QLabel" name="checkerLabel">
      <property name="geometry">
       <rect>
        <x>1004</x>
        <y>0</y>
        <width>121</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>LosAngelous</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="accessBtn">
      <property name="geometry">
       <rect>
        <x>1156</x>
        <y>50</y>
        <width>29</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#accessBtn{
	color: #3686FF;
	line-height: 19px;
	border: none;
	background-color: transparent;
}</string>
      </property>
      <property name="text">
       <string>通过</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rejectBtn">
      <property name="geometry">
       <rect>
        <x>1215</x>
        <y>50</y>
        <width>29</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#rejectBtn{
	color: #FD6A6A;
	line-height: 19px;
	border: none;
	background-color: transparent;
}</string>
      </property>
      <property name="text">
       <string>驳回</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
